kolibri/plugins/learn/assets/test/views/content-card-group-carousel.spec.js

```diff
@@ -152,15 +155,29 @@ describe('ContentCardGroupCarousel', function() {
         });
 
         describe('LTR Languages', function() {
-          it('displays the first content item in the leftmost card');
+          it('displays the first content item in the leftmost card', function() {
+            return Vue.nextTick().then(()=>{
+              const firstCard = this.wrapper.first(card);
```

Comment:
@rtibbles This fails, says that the component isn't found. Tried following the example here:
https://eddyerburgh.gitbooks.io/avoriaz/content/api/selectors.html#vue-components